app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] Foreign key already declared in the _User_ model, and hence no need to declare it again in a referenced model. Simply do something like:
```ruby
# app/models/post.rb
belongs_to :author, class_name: 'User'

Reply:
well resolved! ✔️